tests/cpp_api/unit_tests/README.md

```diff
@@ -4,7 +4,7 @@ This application can be used to verify the functionality of the API offered by r
 
 ## Pre-requisites
 
-* Ubuntu Linux, [version `16.04` or later](https://www.microsoft.com/software-download/windows10)
+* Ubuntu Linux, version `16.04` or later
```

Comment:
we only support the Ubuntu versions ROCm supports, so it should be Ubuntu22 or above